### PR TITLE
List item marker

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| New feature?     | yes
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | #239 
| License          | MIT

### Description

List item marker setting in preference file.

--

### CODE REVIEW

File: src/editor/contentState/updateCtrl.js [hunk 1]
```diff
@@ -118,6 +118,13 @@ const updateCtrl = ContentState => {
     newBlock.listItemType = type
     newBlock.isLooseListItem = preferLooseListItem
 
+    if (type === 'task' || type === 'bullet') {
+      const { bulletListMarker } = this
+      const bulletListItemMarker = marker ? marker.charAt(0) : bulletListMarker
+      newBlock.bulletListItemMarker = bulletListItemMarker
+      console.log(marker.charAt(0))
```
Comment (fxha) [inline src/editor/contentState/updateCtrl.js]: .

File: src/editor/index.js [hunk 2]
```diff
@@ -359,7 +366,7 @@ class Aganippe {
           eventCenter.dispatch('selectionChange', selectionChanges)
           eventCenter.dispatch('selectionFormats', formats)
           this.dispatchChange()
-        }, 1000)
```
Comment (fxha) [inline src/editor/index.js]: If this is the "write" time-out. Why not it set to 100 or 200ms?

File: src/editor/utils/exportMarkdown.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /**
- * Before you edit or update codes in this file, make sure you have read the
+ * Hello contributors, Before you edit or update codes in this file,
```
Comment (fxha) [inline src/editor/utils/exportMarkdown.js]: I would suggest to use `!` or `.` after contributors.

File: src/renderer/components/editor.vue [hunk 4]
```diff
@@ -156,12 +156,18 @@
       this.$nextTick(() => {
         const ele = this.$refs.editor
         const {
-          theme, focus: focusMode, markdown, preferLooseListItem, typewriter,
-          autoPairBracket, autoPairMarkdownSyntax, autoPairQuote
+          theme, focus: focusMode, markdown, preferLooseListItem, typewriter, autoPairBracket, autoPairMarkdownSyntax, autoPairQuote, bulletListMarker
```
Comment (fxha) [inline src/renderer/components/editor.vue]: I think you should split the line - like below.

File: src/editor/contentState/updateCtrl.js (hunk 1 above)
Comment (Jocs) [reply src/editor/contentState/updateCtrl.js]: 🤒

File: src/editor/index.js (hunk 2 above)
Comment (Jocs) [reply src/editor/index.js]: I just want to delay this handler to the next event loop.

File: src/editor/utils/exportMarkdown.js (hunk 3 above)
Comment (Jocs) [reply src/editor/utils/exportMarkdown.js]: ok

File: src/renderer/components/editor.vue (hunk 4 above)
Comment (Jocs) [reply src/renderer/components/editor.vue]: ok

**Pull request merged**